test/coordtest/github-6687.ct

```diff
@@ -0,0 +1,33 @@
+# Regression for https://github.com/MaterializeInc/materialize/issues/6687
```

Comment:
is this the same issue as the one we looked at together?

Reply:
yes. this is the one where a sink created immediately after an index was created would crash. i've added comments here to describe that and how to make it crash.